src/redux/reducers/userReducer.ts

```diff
@@ -29,6 +29,7 @@ export const userReducer = (state: User | null = null, action: UserActions | Orc
         institution: action.user['custom:orgName'],
         application: action.user['custom:application'] as ApplicationName,
         organizationId: getOrganizationIdByOrganizationNumber(action.user['custom:orgNumber']),
+        customerId: action.user['custom:customerId'].split('/').pop(),
```

Comment:
hvis vi vil trenge hele URI til query param for opplsag mot backend senere burde vi ha hele her også

Reply:
men kan godt vente å se hvordan det faktisk blir

Reply:
ja, det er sant. Kan flytte split og pop-en utenfor redux